frontend/management/src/containers/TableHeadContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import {Column} from "../utils/types.tsx";
+import React, {useState} from "react";
+import TableHead from "../components/TableHead.tsx";
+
+function TableHeadContainer({ columns, setOrderBy } : Props) {
+  const [sortField, setSortField] = useState("");
+  const [order, setOrder] = useState("asc");
+  const handleSorting = (accessor) => {
+    const sortOrder = accessor === sortField && order === "asc" ? "desc" : "asc";
```

Comment:
Complex logical expression. Why can't we just assign value of `order`?

Reply:
With this, you don't need to keep track of all sorting orders of each column. I now assume that the column is always ascending at first.

Reply:
Don't we have only one sorting active at a time?

Reply:
if you remove the first part, it continues with the next state of the last ordered column and I want always to start at ascending and then descending.